experiment/run_experiment.py

```diff
@@ -194,7 +195,13 @@ def start_experiment(experiment_name: str, config_filename: str,
                             fuzzer_config)
         # Validate the fuzzer yaml attributes e.g. fuzzer, env, etc.
         validate_fuzzer_config(fuzzer_config)
-        shutil.copy(fuzzer_config, fuzzer_config_dir)
+
+        # Use the hash of the config data as a filename to avoid collisions.
```

Comment:
I think this will break if someone copies a file right?
I think we should include the name of the variant and directory in "fuzzers/" as well, this will make debugging easier and is collision free.

Reply:
Also, can you put this in it's own function?